### DISCUSSION TITLE
GitHub user account suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#118868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I know this is a community forum, and people here cannot help with account related issues.

I'm here to seek guidance on how to proceed, and share my disappointment alongside.

### Back story
I have been a GitHub user for over 8 years now. I've been using GitHub enterprise even at school and work. I've always loved it, and I've pretty much used GH one way or the other every single day.

Exactly a week ago, I decided to research and setup `self-hosted` runners. I loved the experience, so I decided to switch my user profile to an organization that will allow me share runners across the org.
My new profile (created as owner for the org), got flagged without any reason, and after raising a support ticket and reaching out the community, an associate responded that it may have been their automated systems and unflagged it.

The same day, I started a community discussion to better understand sharing runners with `public` vs `private` repos. The next morning, I wake up to my account being suspended. There was only a small time window between my account being unflagged and suspended entirely, during which all I did was to start a discussion.

Link to the discussion (which I believe has been removed/inaccessible): https://github.com/orgs/community/discussions/118477

There was no email informing "why" my account was suspended or which ToS I violated. I've been researching ever since, only to find many others sharing the same frustration.

I'm incredibly disappointed and heart broken with the recent experience. I use GitHub enterprise at work EVERY SINGLE DAY, I can't stop wondering how come a user profile gets flagged one day, and suspended the next with absolutely no form of communication.

Since there was ~12 hours between my account being unflagged and suspended entirely, I couldn't even add any other members to my organization. I have more than 30 private repositories in the org, where I have some very important work stored, which is also my daily reference.

I created a support ticket #2723183, for which I still haven't had any response. The only user profile that has access to my organization has been suspended blocking me from accessing my work.

The more I read about relevant issues (on top of which, there is zero communication from the support team), the more I'm scared that I might permanently loose access to my org.

Now, when I try to login, even the `can't sign in` verification code (that's valid only for 20 minutes) takes 2 hours to reach my inbox.

### Help Needed
Given the current state of things, how do I proceed?
Do I just sit tight and pray GitHub's support team responds? Is there an alternate channel to reach out to them?
I know paid accounts get priority, but my 7 year old account has been converted to an organization.
Is there a way I can just verify my identity and get the private repos from it?

### Related Information
Support ticket: #2723183
Organization: @thevickypedia
Suspended user profile: @dormant-user

Any help or guidance this regard will be much appreciated. Thank you in advance.

**Comment by ettaboyle** [comment]
Hi @rustic-monkey 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, as I can see you already know, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
